cmake/CMakeLists.txt

```diff
@@ -3,12 +3,14 @@
 # Licensed under the MIT License.
 
 # Minimum CMake required
-cmake_minimum_required(VERSION 3.13)
+cmake_minimum_required(VERSION 3.18)
```

Comment:
>3.18 [](http://example.com/codeflow?start=31&length=4)

nice! just curious, could we go right to 3.20?

Reply:
This is minimum version required. Yes we can increase it to 3.20 but it doesn't give us benefits. I think we can consider to update it when there is a need.